### PR TITLE
fix Sonarqube issues

### PR DESCRIPTION
**Pull request by taseakpoexternal**
Resolve issues with SonarQube
Not related to a user story

### CODE REVIEW

File: src/EPR.PRN.Backend.API/Controllers/PrnController.cs [hunk 1]
```diff
@@ -37,22 +37,22 @@ public async Task<IActionResult> GetPrn([FromHeader(Name = "X-EPR-ORGANISATION")
...
+        {
             if (orgId == Guid.Empty)
                 return Unauthorized();
 
             var result = await _prnService.GetSearchPrnsForOrganisation(orgId, request);
-            
+
             return Ok(result);
-		}
+        }
```
Comment (saywasingh-eviden) [inline src/EPR.PRN.Backend.API/Controllers/PrnController.cs:53]: Please remove these changes related to PRN 1 's code. I think its a bit a confusing.

File: src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ public IEnumerable<Materials> GetAllMaterials()
             return GetMaterialsFromDatabase();
         }
 
-        private IEnumerable<Materials> GetMaterialsFromDatabase()
+        private static List<Materials> GetMaterialsFromDatabase()
```
Comment (saywasingh-eviden) [inline src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs:13]: Why are we switching to List rather than IEnumerable?

Comment (taseakpoexternal) [reply src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs:13]: better for performance

File: src/EPR.PRN.Backend.API/Controllers/PrnController.cs (hunk 1 above)
Comment (taseakpoexternal) [reply src/EPR.PRN.Backend.API/Controllers/PrnController.cs:53]: this is just auto formatting there isnt any code change. Think visual studio does it on save. Maybe more of a hassle to revert if the formatting is correct?

File: src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs (hunk 2 above)
Comment (harishpaladugu13) [reply src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs:13]: List inherits IEnumerable.
we choose what to use depending on what to do.

Comment (saywasingh-eviden) [reply src/EPR.PRN.Backend.Data/Repositories/MaterialRepository.cs:13]: I don't agree its better performance but as discussed if SQ is complaining is wanting this change then we'll have to go with it.

**Review by saywasingh-eviden — APPROVED** [review]

**Review by harishpaladugu13 — APPROVED** [review]

**Pull request merged**